variables.tf

```diff
@@ -379,6 +418,27 @@ variable "os_disk_type" {
   nullable    = false
 }
 
+variable "outbound_ip_address_ids" {
+  type        = list(any)
```

Comment:
According to the [provider code](https://github.com/hashicorp/terraform-provider-azurerm/blob/307e6478cf507c41301413475d77574618031db5/internal/services/containers/kubernetes_cluster_resource.go#L828-L838), the type should be `set(string)` here.